docs/design/diagrams/invoke_sequence.puml

```diff
@@ -1,8 +1,53 @@
 @startuml
 
-participant "Provider"
-participant "Consumer"
+autonumber
 
-"Consumer" -> "Provider" : Invoke - request (entity id, consumer uri, response id, payload)
+"Digital Twin Consumer" -> "In-Vehicle Digital Twin Service" : FindById("dtmi:sdv:HMI:ShowNotification;1") - request
+"Digital Twin Consumer" <- "In-Vehicle Digital Twin Service" : FindById - response
+note left
+entityAccessInfo:
+{
+    name: "ShowNotification"
+    id: "dtmi:sdv:HMI:ShowNotifiation;1"
+    description: "..."
+    endpointInfoList: [
+        {
+            protocol: "grpc"
+            operationss: ["Invoke"]
+            uri: "http://127.0.0.1:4000"
+            context: "dtmi:sdv:HMI:ShowNotifiation;1"
+        }
+    ]
+}
+end note
+
+"Digital Twin Consumer" -> "Digital Twin Provider" : Invoke - request
+note right
+{
+    entity_id: "dtmi:sdv:HMI:ShowNotification:request;1"
+    consumer_uri: "http://127.0.0.1:5000"
+    response_id: "123456789"
+    payload:
```

Comment:
Should the **value** field for Get, Set and Subscribe requests be renamed to **payload** to be consistent?

Reply:
It's okay for now.  We may change this later.